### PR TITLE
ci: testing with PT 2.0.0

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

as PT 2.0.0 is out we shall test it

<details>
  <summary>Before submitting</summary>

- [ ] Was this **discussed/agreed** via a Github issue? (no need for typos and docs improvements)
- [ ] Did you read the [contributor guideline](https://github.com/Lightning-AI/metrics/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure to **update the docs**?
- [ ] Did you write any new **necessary tests**?

</details>

<details>
  <summary>PR review</summary>

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

</details>

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: .azure/gpu-pipeline.yml [hunk 1]
```diff
@@ -24,10 +24,15 @@ jobs:
           docker-image: 'pytorch/pytorch:1.9.0-cuda11.1-cudnn8-runtime'
           agent-pool: 'lit-rtx-3090'
           torch-ver: '1.9.0'
-        'PyTorch - stable':
+        'PyTorch - LTS':
```
Comment (akihironitta) [inline .azure/gpu-pipeline.yml:27]: nit: LTS sounds a bit confusing to me. Maybe `PyTorch - 1.13` or `PyTorch - 1`?

**Review by akihironitta — APPROVED** [review]

**Comment by Borda** [comment]
Interesting error on Windows... @SkafteNicki 
```
FAILED unittests\wrappers\test_tracker.py::test_best_metric_for_not_well_defined_metric_collection[base_metric0] - RuntimeError: a Tensor with 9 elements cannot be converted to Scalar
FAILED unittests\wrappers\test_tracker.py::test_best_metric_for_not_well_defined_metric_collection[base_metric1] - RuntimeError: a Tensor with 9 elements cannot be converted to Scalar
FAILED unittests\wrappers\test_tracker.py::test_metric_tracker_and_collection_multioutput[input_to_tracker0-Tensor] - RuntimeError: a Tensor with 2 elements cannot be converted to Scalar
```

File: .azure/gpu-pipeline.yml (hunk 1 above)
Comment (akihironitta) [reply .azure/gpu-pipeline.yml:27]: You're right! just wanted to make it look beginner-friendly and less confusing :)

File: requirements.txt [hunk 2]
```diff
@@ -2,7 +2,7 @@
 #  in case you want to preserve/enforce restrictions on the latest compatible version, add "strict" as an in-line comment
 
 numpy >=1.17.2
-torch >=1.8.1, <=1.13.1
+torch >=1.8.1, <=2.0.0
```
Comment (stancld) [inline requirements.txt:5]: Shouldn't we also bump max version for torchvision?

Comment (Borda) [reply requirements.txt:5]: good catch :)

**Review by stancld — APPROVED** [review]

**Comment by SkafteNicki** [comment]
> Interesting error on Windows... @SkafteNicki 
> ```
> FAILED unittests\wrappers\test_tracker.py::test_best_metric_for_not_well_defined_metric_collection[base_metric0] - RuntimeError: a Tensor with 9 elements cannot be converted to Scalar
> FAILED unittests\wrappers\test_tracker.py::test_best_metric_for_not_well_defined_metric_collection[base_metric1] - RuntimeError: a Tensor with 9 elements cannot be converted to Scalar
> FAILED unittests\wrappers\test_tracker.py::test_metric_tracker_and_collection_multioutput[input_to_tracker0-Tensor] - RuntimeError: a Tensor with 2 elements cannot be converted to Scalar
> ```

I think I know what is wrong, will fix later today when I got time 😊

**Review by SkafteNicki — APPROVED** [review]

**Comment by ddelange** [comment]
hi @Borda 👋

when do you think you could cut out a new release containing this PR?

**Comment by Borda** [comment]
> when do you think you could cut out a new release containing this PR?

we can do RC today :)

**Comment by ddelange** [comment]
many thanks for the quick release!

**Comment by ddelange** [comment]
can happily report the rc is working well on cp311 with pt2.0 🎉

Two edits were necessary:
- https://github.com/autogluon/autogluon/pull/3190/commits/6e626af6a2ce70b3255ea495b07ae12325a7e92d
- https://github.com/autogluon/autogluon/pull/3190/commits/cdbd78e837083b98bc1e866a929924237880e383

**Pull request merged**